### ISSUE TITLE
Ignoring comments in .vimspector.json

### ISSUE THREAD

**Issue by mark2185** [ilyachur/cmake4vim#164]
**Is your feature request related to a problem? Please describe.**
`vimspector` ignores comments in JSON (`// comment`), but `cmake4vim` doesn't so it complains about parsing the JSON file when trying to run a target.

**Describe the solution you'd like**
Ignore the content after `//`, but don't delete it when writing `.vimspector.json` again.

**Describe alternatives you've considered**
Not writing comments.

**Comment by ilyachur** [comment]
Unfortunately json standard doesn't support comments.

> As per Standard JSON rules, There is no official support for Comments in JSON content.

I read `vimspector` config to json dictionary and write it also as json, so in case of C style comments it returns error

**Comment by mark2185** [comment]
I know,  but vimspector _explicitly_ [supports C styled comments](https://github.com/puremourning/vimspector#debug-profile-configuration)

>  The JSON configuration file allows C-style comments:
>
>    // comment to end of line ...
>    /* inline comment ... */

So we do have `vimspector` support, but we also have a missing feature.

**Comment by ilyachur** [comment]
Yes I know, honestly I don't know the simple way how to support comments. Looks like for that I need to change the system of reading/writing vimspector config file.
